external/st_audiofile/src/st_audiofile.c

```diff
@@ -449,6 +496,25 @@ uint64_t st_read_s16(st_audio_file* af, int16_t* buffer, uint64_t count)
     case st_audio_file_mp3:
         count = drmp3_read_pcm_frames_s16(af->mp3, count, buffer);
         break;
+    case st_audio_file_wv:
+        {
+            uint32_t channels = af->cache.wv.channels;
+            int32_t* buf_i32 = (int32_t*)malloc(4 * channels * count);
```

Comment:
May I change this for some C++-y lifetime management version?

Reply:
Yes sure, that would be great.

Reply:
Nevermind I forgot this is a C file.